modules/managed-cluster-remove-notification-contacts.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="remove-notification-contact_{context}"]
+= Removing notification contacts from your cluster
+
+Notification contacts receive emails when cluster notifications are sent to the cluster.
+
+You can remove notification contacts in your cluster support settings to prevent them from receiving notification emails.
+
+.Prerequisites
+* Your cluster is deployed and registered to the {hybrid-console}.
+* You are logged in to the {hybrid-console-second}.
+
+.Procedure
+. Navigate to the Clusters page of the {hybrid-console-second}.
+. Click the name of your cluster to go to the cluster details page.
+. Click the **Support** tab.
+. On the **Support** tab, find the **Notification contacts** section and locate the email address of the recipient you want to remove.
+. Click **Remove recipient**.
+. Click **Save**.
```

Comment:
@laubai These two steps seems wrong. You have three vertical dots associated to the user and once you click the same, the context menu "Delete" will show for you. Click action on "Delete" context menu will be removed the user from the notification contacts. 
![image](https://github.com/openshift/openshift-docs/assets/93653094/d560982f-d25e-4c70-b696-91de30e4ec1d)

Reply:
Thankyou! Fixed this, not sure why the interface showed so differently for me... might have been on a stage cluster